.gitignore

```diff
@@ -1,3 +1,4 @@
 .vscode/
 dotm.code-workspace
 .DS_Store
+extracted/
```

Comment:
Good idea putting in the "extracted" here.  I totally forgot this... :woman_facepalming:

Reply:
Even so, we both have added numerous extra files that we probably shouldn't have...  Is there a way to clean this up?  I dunno - maybe re-zip the files or something?